Failed find: Maximum call stack size exceeded

vsts-task-lib 2.4.0
My task is looking for a file produced during the build. I know the name of the file (report-task.txt), but not its location. So I'm using:
```
tl.findMatch(
  tl.getVariable('Agent.BuildDirectory'),
  path.join('**', 'report-task.txt')
);
```
Unfortunately, when there are too many files in the build directory, it fails with:
```
2018-09-20T09:14:03.3401724Z ##[debug]Agent.BuildDirectory=C:\vsts-agent\agent\_work\2
2018-09-20T09:14:03.3401908Z ##[debug]defaultRoot: 'C:\vsts-agent\agent\_work\2'
2018-09-20T09:14:03.3402086Z ##[debug]findOptions.followSpecifiedSymbolicLink: 'true'
2018-09-20T09:14:03.3402256Z ##[debug]findOptions.followSymbolicLinks: 'true'
2018-09-20T09:14:03.3402421Z ##[debug]matchOptions.debug: 'false'
2018-09-20T09:14:03.3402581Z ##[debug]matchOptions.nobrace: 'true'
2018-09-20T09:14:03.3402741Z ##[debug]matchOptions.noglobstar: 'false'
2018-09-20T09:14:03.3402899Z ##[debug]matchOptions.dot: 'true'
2018-09-20T09:14:03.3403057Z ##[debug]matchOptions.noext: 'false'
2018-09-20T09:14:03.3403299Z ##[debug]matchOptions.nocase: 'true'
2018-09-20T09:14:03.3403458Z ##[debug]matchOptions.nonull: 'false'
2018-09-20T09:14:03.3403624Z ##[debug]matchOptions.matchBase: 'false'
2018-09-20T09:14:03.3403783Z ##[debug]matchOptions.nocomment: 'false'
2018-09-20T09:14:03.3403946Z ##[debug]matchOptions.nonegate: 'false'
2018-09-20T09:14:03.3404108Z ##[debug]matchOptions.flipNegate: 'false'
2018-09-20T09:14:03.3404265Z ##[debug]pattern: '**\report-task.txt'
2018-09-20T09:14:03.3408591Z ##[debug]findPath: 'C:\vsts-agent\agent\_work\2'
2018-09-20T09:14:03.3408807Z ##[debug]statOnly: 'false'
2018-09-20T09:14:03.3410070Z ##[debug]findPath: 'C:\vsts-agent\agent\_work\2'
2018-09-20T09:14:03.3410294Z ##[debug]findOptions.followSpecifiedSymbolicLink: 'true'
2018-09-20T09:14:03.3410479Z ##[debug]findOptions.followSymbolicLinks: 'true'
2018-09-20T09:14:03.3413368Z ##[debug]  C:\vsts-agent\agent\_work\2 (directory)
[...] (many files)
2018-09-20T09:14:09.9229553Z ##[debug][SQ] Publish task error: Failed find: Maximum call stack size exceeded
2018-09-20T09:14:09.9229867Z ##[debug]task result: Failed
2018-09-20T09:14:09.9230850Z ##[error]Failed find: Maximum call stack size exceeded
2018-09-20T09:14:09.9231251Z ##[debug]Processed: ##vso[task.issue type=error;]Failed find: Maximum call stack size exceeded
2018-09-20T09:14:09.9231740Z ##[debug]Processed: ##vso[task.complete result=Failed;]Failed find: Maximum call stack size exceeded
```

Would it be possible to fix the behavior?

--- a/node/package.json
+++ b/node/package.json
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-task-lib",
-  "version": "3.0.0-preview",
+  "version": "3.1.0-preview",
   "description": "Azure Pipelines Task SDK",
   "main": "./task.js",
   "typings": "./task.d.ts",

--- a/node/task.ts
+++ b/node/task.ts
@@ -918,7 +918,9 @@ export function find(findPath: string, options?: FindOptions): string[] {
                 let childItems: _FindItem[] =
                     fs.readdirSync(item.path)
                         .map((childName: string) => new _FindItem(path.join(item.path, childName), childLevel));
-                stack.push(...childItems.reverse());
+                for (var i = childItems.length - 1; i >= 0; i--) {
+                    stack.push(childItems[i]);
+                }
             }
             else {
                 debug(`  ${item.path} (file)`);

